forwards_paper.tex

```diff
@@ -221,18 +276,22 @@ \section*{Introduction}
 
 In this paper, we describe a storage method for \emph{succinct tree sequences}
 (and hence, genome sequence) as well as an algorithm for simplifying these.
-The data structure is \emph{succinct} in the sense that its' space usage is close to optimal,
-while still allowing efficient retrieval of information (see, e.g.,
-\citet{gog2014theory}).
-The storage method and simplification algorithm are both implemented
-as improvements to the algorithmic tools and data structures in \msprime{}.
+The data structure is \emph{succinct} in the sense that its space usage is close to optimal,
+while still allowing efficient retrieval of information (see, e.g., \citet{gog2014theory}).
 We also describe how these tools can efficiently record,
 and later process, the embellished population pedigree from a forwards-time simulation.
-While providing substantial savings in computational time and space, our methods provide much
-more information than simply simulating the genomes -- the tree sequence
+While providing substantial savings in computational time and space, our methods provide 
+in principle much more information than simply simulating the genomes -- the tree sequence
 encodes all marginal genealogies of individuals living at the end of the simulation.
+These marginal genealogies enable fast data storage and processing,
+but also provide additional information that can be used to better understand
+the notoriously complex dynamics of population genetics. \revpoint{1}{4}
 Although we were motivated by a need for more efficient genomic simulations,
 these tools may prove more widely useful.
+This work originated as improvements to the algorithmic tools and data structures 
+in the coalescent simulator \msprime{}; \linelabel{msprime_rel}
+the tools are currently available bundled with the Python package \msprime{},
+but will soon be separately available as a Python package named \tskit{}.
```

Comment:
"available as a package name \tskit{}, which will include a Python API and an embeddable C library."